### ISSUE TITLE
1.1.7 submission autochecks: isbn and unregistered prints

### ISSUE THREAD

**Issue by swihart** [swihart/repeated#21]
```
* using log directory ‘/srv/hornik/tmp/CRAN/repeated.Rcheck’
* using R Under development (unstable) (2023-08-07 r84885)
* using platform: x86_64-pc-linux-gnu
* R was compiled by
    Debian clang version 16.0.6 (6)
    GNU Fortran (Debian 13.2.0-1) 13.2.0
* running under: Debian GNU/Linux trixie/sid
* using session charset: UTF-8
* checking for file ‘repeated/DESCRIPTION’ ... OK
* this is package ‘repeated’ version ‘1.1.7’
* package encoding: UTF-8
* checking CRAN incoming feasibility ... [4s/5s] NOTE
Maintainer: ‘Bruce Swihart <bruce.swihart@gmail.com>’

Found the following (possibly) invalid URLs:
  URL: isbn-10:0198505590
    From: DESCRIPTION
    Message: Invalid URI scheme
  URL: isbn-10:0198508123
    From: DESCRIPTION
    Message: Invalid URI scheme
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking for sufficient/correct file permissions ... OK
* checking serialization versions ... OK
* checking whether package ‘repeated’ can be installed ... [17s/17s] OK
* used C compiler: ‘Debian clang version 16.0.6 (6)’
* used Fortran compiler: ‘GNU Fortran (Debian 13.2.0-1) 13.2.0’
* checking package directory ... OK
* checking for future file timestamps ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... [0s/0s] OK
* checking whether the package can be loaded with stated dependencies ... [0s/0s] OK
* checking whether the package can be unloaded cleanly ... [0s/0s] OK
* checking whether the namespace can be loaded with stated dependencies ... [0s/0s] OK
* checking whether the namespace can be unloaded cleanly ... [0s/0s] OK
* checking loading without being on the library search path ... [0s/0s] OK
* checking use of S3 registration ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... NOTE
Mismatches for apparent methods not registered:
print:
  function(x, ...)
print.binnest:
  function(z)

print:
  function(x, ...)
print.summary.glmm:
  function(z, ...)

print:
  function(x, ...)
print.glmm:
  function(z, ...)

print:
  function(x, ...)
print.bivbinom:
  function(z)

print:
  function(x, ...)
print.bevbetab:
  function(z)
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... [21s/21s] OK
* checking Rd files ... [0s/0s] OK
* checking Rd metadata ... OK
* checking Rd line widths ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking line endings in C/C++/Fortran sources/headers ... OK
* checking pragmas in C/C++ headers and code ... OK
* checking compilation flags used ... OK
* checking compiled code ... OK
* checking examples ... [2s/2s] OK
* checking PDF version of manual ... [3s/3s] OK
* checking HTML version of manual ... [1s/1s] OK
* checking for non-standard things in the check directory ... OK
* checking for detritus in the temp directory ... OK
* DONE
Status: 2 NOTEs
```

**Comment by swihart** [comment]
Well after doing:

```
cran <- tools::CRAN_package_db()
desc_with_isbn <- grep("isbn", cran$Description, value = TRUE)


```

I see that maybe I should switch the `-` and the `:`.  So let's try that.

**Comment by swihart** [comment]
So from [SO](https://stackoverflow.com/a/71308978/2727349):


    * use @export by itself without naming the function (as suggested by @KonradRudolph)
    * remove the @usage line
    * use is.nan(df) in the example (as I suggested)

So I'll remove `binnest` from the `@export binnest` ... there is no usage line...well, since using roxygen, I did another cmd+shft+D just to see if usage would change...and as far as I can tell there is no print.binnest being used in examples.

However, I looked more closely at NAMESPACE and I have print methods for gar, etc that seem to be fine and don't appear in the checks of the OP checks that opened this issue.  I investigated and turns out that I have an `@export` right above the print.gar -- so I am going to undo what I did (that is, have `@export binnest` right above the `binnest` function) and then put an `@export` right before `print.binnest`.  the Namespace should update after a cmd-shft-D.  I also will do the `z <- x` trick I had to do when I first got this package on CRAN for the first time since R3.0.

basically, make it look like this and do a cmd-shft-D

```
#' @export
print.binnest <- function(x, ...){
  z <- x
```

**Comment by swihart** [comment]
Ok, I submitted and got through those prelim checks.  Now, I just sent reply-all email stating I fixed the mixed free- and fixed-format fortran and touched up the examples that was making M1 error-out.  Let's see where it goes.  I hope I avoid archival...
